### PR TITLE
[DASH-2461] Remove the possibility to add/remove hosting in the Dashb…

### PR DESCRIPTION
**Pull request by tomsonpl**
…oard

At the moment I just deleted all add/delete buttons. Do you think we should leave EDIT option? 
screen shot 2017-01-18 at 16 57 07
screen shot 2017-01-18 at 16 56 56
screen shot 2017-01-18 at 16 56 50

### CODE REVIEW

File: src/apps/Hosting/HostingList.js [hunk 1]
```diff
@@ -142,6 +140,7 @@ const HostingList = React.createClass({
           docsUrl="http://docs.syncano.io/docs"
           handleClick={HostingActions.showDialog}
           buttonLabel="Add Hosting"
```
Comment (Flixow) [inline src/apps/Hosting/HostingList.js]: Need we this props if button is hidden?

File: src/common/EmptyView/EmptyView.js [hunk 2]
```diff
@@ -13,6 +13,7 @@ const EmptyView = ({
   docsUrl,
   description,
   urlLabel,
+  shouldBeHidden,
```
Comment (Flixow) [inline src/common/EmptyView/EmptyView.js]: Button shouldn't be render if buttonLabel props is undefined. We don't need new one.

File: src/apps/Hosting/HostingList.js (hunk 1 above)
Comment (tomsonpl) [inline src/apps/Hosting/HostingList.js]: ofc we don't need this :) you are right

File: src/common/EmptyView/EmptyView.js (hunk 2 above)
Comment (tomsonpl) [inline src/common/EmptyView/EmptyView.js]: But we change this in just one zerostate, so we would have to adjust all others if we choose your suggestion.   Am I wrong?

Comment (tomsonpl) [inline src/common/EmptyView/EmptyView.js]: ah now I got what you meant.   I deleted button label , the text went away. but the button stayed. that is why I added a new props.

**Comment by gh3r** [comment]
@tomsonpl ???

**Comment by tomsonpl** [comment]
LGTM, thanks for tests @gh3r

**Pull request merged**